temp/OnnxTransformTempSolution/Microsoft.ML.Transforms.Onnx/OnnxTransform.cs

```diff
@@ -0,0 +1,294 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Scoring;
+using Microsoft.ML.Transforms.Onnx;
+using OnnxShape = System.Collections.Generic.List<long>;
+
+[assembly: LoadableClass(OnnxTransform.Summary, typeof(IDataTransform), typeof(OnnxTransform.Arguments), typeof(SignatureDataTransform),
+    OnnxTransform.UserName, OnnxTransform.ShortName)]
+
+// This is for de-serialization from a binary model file.
+[assembly: LoadableClass(typeof(OnnxTransform.OnnxMapper), null, typeof(SignatureLoadRowMapper),
+    "", OnnxTransform.OnnxMapper.LoaderSignature)]
+
+namespace Microsoft.ML.Transforms.Onnx
+{
+    public static class OnnxTransform
+    {
+        public sealed class Arguments : TransformInputBase
+        {
+
+            [Argument(ArgumentType.Required, HelpText = "Path to the onnx model file.", ShortName = "model", SortOrder = 0)]
+            public string ModelFile;
+
+            [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "TBD", SortOrder = 1)]
+            public string InputColumn;
+
+            [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "TBD", SortOrder = 2)]
+            public string OutputColumn;
+
+            public OnnxModelInfo ModelInfo;
+        }
+
+        public const string Summary = "Transforms the data using the Onnx model.";
+        public const string UserName = "OnnxTransform";
+        public const string ShortName = "Onnx";
+        private const string RegistrationName = "OnnxTransform";
+
+        /// <summary>
+        /// Convenience constructor for public facing API.
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="input">Input <see cref="IDataView"/>. This is the output from previous transform or loader.</param>
+        /// <param name="modelFile">This is the frozen Onnx model file. https://www.tensorflow.org/mobile/prepare_models </param>
```

Comment:
>https://www.tensorflow.org/mobile/prepare_models [](start = 72, length = 48)

This needs to be updated. The term "frozen" also is only for TF models, isn't it?